src/jit/block.cpp

```diff
@@ -232,38 +233,126 @@ void BasicBlock::dspBlockILRange()
-    if (bbFlags & BBF_DONT_REMOVE)          printf("keep ");
-    if (bbFlags & BBF_IMPORTED)             printf("i ");
-    if (bbFlags & BBF_INTERNAL)             printf("internal ");
-    if (bbFlags & BBF_FAILED_VERIFICATION)  printf("failV ");
-    if (bbFlags & BBF_TRY_BEG)              printf("try ");
-    if (bbFlags & BBF_NEEDS_GCPOLL)         printf("poll ");
-    if (bbFlags & BBF_RUN_RARELY)           printf("rare ");
-    if (bbFlags & BBF_LOOP_HEAD)            printf("Loop ");
-    if (bbFlags & BBF_LOOP_CALL0)           printf("Loop0 ");
-    if (bbFlags & BBF_LOOP_CALL1)           printf("Loop1 ");
-    if (bbFlags & BBF_HAS_LABEL)            printf("label ");
-    if (bbFlags & BBF_JMP_TARGET)           printf("target ");
-    if (bbFlags & BBF_HAS_JMP)              printf("jmp ");
-    if (bbFlags & BBF_GC_SAFE_POINT)        printf("gcsafe ");
-    if (bbFlags & BBF_FUNCLET_BEG)          printf("flet ");
-    if (bbFlags & BBF_HAS_INDX)             printf("idx ");
-    if (bbFlags & BBF_HAS_NEWARRAY)         printf("new[] ");
-    if (bbFlags & BBF_HAS_NEWOBJ)           printf("newobj ");
+    if (bbFlags & BBF_VISITED)
+    {
+        printf("v ");
+    }
+    if (bbFlags & BBF_MARKED)
+    {
+        printf("m ");
+    }
+    if (bbFlags & BBF_CHANGED)
+    {
+        printf("! ");
+    }
+    if (bbFlags & BBF_REMOVED)
+    {
+        printf("del ");
+    }
+    if (bbFlags & BBF_DONT_REMOVE)
+    {
+        printf("keep ");
+    }
+    if (bbFlags & BBF_IMPORTED)
+    {
+        printf("i ");
+    }
+    if (bbFlags & BBF_INTERNAL)
+    {
+        printf("internal ");
+    }
+    if (bbFlags & BBF_FAILED_VERIFICATION)
+    {
+        printf("failV ");
+    }
+    if (bbFlags & BBF_TRY_BEG)
+    {
+        printf("try ");
+    }
+    if (bbFlags & BBF_NEEDS_GCPOLL)
+    {
+        printf("poll ");
+    }
+    if (bbFlags & BBF_RUN_RARELY)
+    {
+        printf("rare ");
+    }
+    if (bbFlags & BBF_LOOP_HEAD)
+    {
+        printf("Loop ");
+    }
+    if (bbFlags & BBF_LOOP_CALL0)
+    {
+        printf("Loop0 ");
+    }
+    if (bbFlags & BBF_LOOP_CALL1)
+    {
+        printf("Loop1 ");
+    }
+    if (bbFlags & BBF_HAS_LABEL)
+    {
+        printf("label ");
+    }
+    if (bbFlags & BBF_JMP_TARGET)
+    {
+        printf("target ");
+    }
+    if (bbFlags & BBF_HAS_JMP)
+    {
+        printf("jmp ");
+    }
+    if (bbFlags & BBF_GC_SAFE_POINT)
+    {
+        printf("gcsafe ");
+    }
+    if (bbFlags & BBF_FUNCLET_BEG)
+    {
+        printf("flet ");
+    }
+    if (bbFlags & BBF_HAS_INDX)
+    {
+        printf("idx ");
+    }
+    if (bbFlags & BBF_HAS_NEWARRAY)
+    {
+        printf("new[] ");
+    }
+    if (bbFlags & BBF_HAS_NEWOBJ)
+    {
+        printf("newobj ");
+    }
 #if FEATURE_EH_FUNCLETS && defined(_TARGET_ARM_)
-    if (bbFlags & BBF_FINALLY_TARGET)       printf("ftarget ");
+    if (bbFlags & BBF_FINALLY_TARGET)
+        printf("ftarget ");
```

Comment:
We have only run x64. This if is under a TARGET_ARM ifdef, so it didn't get modified due to clang-tidy running on preprocessed code.